modules/asg/main.tf

```
@@ -50,13 +45,12 @@ resource "aws_security_group" "asg_sg" {
 resource "aws_launch_template" "t3micro" {
   name_prefix            = "t3micro"
   image_id               = data.aws_ami.amazon-linux-2.id
-  instance_type          = "t3.micro"
-  key_name               = "MyKeyPair"
+  instance_type          = "t3.micro"                       #Removed key variable
```

No need to have this comment
```suggestion
  instance_type          = "t3.micro"
```

---

resolved

---

Still not fixed